odoo/addons/base/models/res_lang.py

```diff
@@ -88,15 +88,24 @@ def _register_hook(self):
         if not self.search_count([]):
             _logger.error("No language is active.")
 
-    @api.model
+    # TODO remove me after v14
     def load_lang(self, lang, lang_name=None):
-        """ Create the given language if necessary, and make it active. """
-        # if the language exists, simply make it active
-        language = self.with_context(active_test=False).search([('code', '=', lang)], limit=1)
-        if language:
-            language.write({'active': True})
-            return language.id
+        _logger.warning("Call to deprecated method load_lang, use _create_lang or _activate_lang instead")
+        language = self._activate_lang(lang)
+        if not language:
+            self._create_lang(lang, lang_name=lang_name)
+
+    def _activate_lang(self, code):
+        """ Activate languages
+        :param code: code of the language to activate
+        :return: the language matching 'code' activated
+        """
+        lang = self.with_context(active_test=False).search([('code', '=', code)])
+        lang.filtered(lambda l: not l.active).write({'active': True})
```

Comment:
The `code` is unique, so by construction `lang` contains at most 1 record.
What about a plain stupid `if` statement like
```suggestion
        if lang and not lang.active:
            lang.active = True
```